File: MODUL 5/SOAL 4/PRAK404-2210817210026-Zulfa Auliya Akbar.c
#include <stdio.h>

int main(){

    float n1,n2,pil;

    while(pil!=5){
        printf("Pilih program \n1.Penjumlahan \n2.Pengurangan \n3.perkalian \n4.Pembagian\n5.Exit \nMasukan Pilihan :");
        scanf("%f",&pil);

        if(pil!=1 && pil!=2 && pil!=3 && pil!=4 && pil!=5){
            printf("Input anda salah, silahkan coba lagi \n \n");
        }
        else if (pil == 5){
            printf("Terimakasih, telah menggunakan kalkulator Zulfa Auliya Akbar \n \n");
        }
        else {
            printf("Masukan nilai pertama :");
            scanf("%f",&n1);
            printf("Masukan Nilai Kedua :");
            scanf("%f",&n2);

            if(pil==1){
                printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f \n \n",n1,n2,n1+n2);
            }
            else if(pil==2){
                printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f \n \n",n1,n2,n1-n2);
            }
            else if(pil==3){
                printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f \n \n",n1,n2,n1*n2);
            }
            else if(pil==4){
                printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f \n \n",n1,n2,n1/n2);
            }
        }
    }
}

File: MODUL 5/SOAL 4/PRAK404-2210817210026-Zulfa Auliya Akbar.py
pil = 0
while (pil !=5):
    print("Pilih program \n1.Penjumlahan \n2.Pengurangan \n3.perkalian \n4.Pembagian\n5.Exit \nMasukan Pilihan :" )
    pil = int(input())

    if pil!=1 and pil!=2 and pil!=3 and pil!=4 and pil!=5:
        print("Input anda salah, silahkan coba lagi \n")
    
    elif pil == 5:
        print("Terimakasih, telah menggunakan kalkulator Zulfa Auliya Akbar \n")
    
    else:
        n1=float(input("Masukan nilai pertama :"))
        n2=float(input("Masukan nilai kedua :"))

        if pil==1:
            print("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f \n" %(n1,n2,n1+n2))
        if pil==2:
            print("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f \n" %(n1,n2,n1-n2))
        if pil==3:
            print("Hasil perkalian antara %.2f dengan %.2f adalah %.2f \n" %(n1,n2,n1*n2))
        if pil==4:
            print("Hasil pembagian antara %.2f dengan %.2f adalah %.2f \n" %(n1,n2,n1/n2))
